lib/tasks/govuk_assets.rake

```diff
@@ -16,4 +16,27 @@ namespace :govuk_assets do
     status = result.successful? ? 'OK' : 'Error'
     puts "#{status}: #{result.written_count} documents updated"
   end
+
+  desc 'Unreplace an asset and delete the replacement'
+  task :unreplace_asset, [:uuid] => :environment do |_, args|
+    begin
+      uuid = args[:uuid]
+      asset = Asset.unscoped.find_by(uuid: uuid)
```

Comment:
If you use `find_by!` it can raise an exception if no asset is found and we avoid the subsequent `if` block.